domain/model/bootstrap/bootstrap.go

```diff
@@ -74,6 +74,7 @@ func CreateModel(
 			return fmt.Errorf("invalid model uuid: %w", err)
 		}
 
+		finaliser := state.Finalise()
```

Comment:
Should the state method be called `GetFinaliser`. As it is named now, it implies that it is actually doing to finalise operation when called, and that's not the case.

Reply:
Done & Done